src/app/components/SwapStatusCard.jsx

```diff
@@ -23,10 +24,10 @@ const StatusFooter = ({ className, children, ...props }) => (
 const priceChange = (timestamp, asset) => {
   const { change1, change7d, change24 } = asset
   const daysSinceTrade = (Date.now() - timestamp) / 86400000
-  const timespan = daysSinceTrade < 1 ? 'Last 24hr: ' : daysSinceTrade >= 2 ? 'Last 7d: ' : 'Last 1d: '
-  var priceChangeText = daysSinceTrade < 1 ? change1 : daysSinceTrade >= 2 ? change7d : change24
+  const timespan = daysSinceTrade <= .0417 ? 'Last 1hr: ' : daysSinceTrade >= 2 ? 'Last 7d: ' : 'Last 24hrs: '
```

Comment:
I think it would be clearer if you used hoursSinceTrade rather than days so you don't need to use .0417